contracts/interchain-token-service/tests/executable.rs

```diff
@@ -84,8 +91,8 @@ mod test {
             token_id: BytesN<32>,
             token_address: Address,
             amount: i128,
-        ) {
-            Self::validate(env);
+        ) -> Result<(), ContractError> {
+            let _ = Self::validate(env);
 
```

Comment:
Maybe return a custom error here on some condition (e.g. amount being less than some value etc.) to demonstrate usage and test the error